### PR TITLE
Disabled "enter" key when using forms and changed options for feedback

### PR DESCRIPTION
**Pull request by GrantComm**
Screenshot 2020-08-04 at 1 29 48 PM

Screenshot 2020-08-05 at 1 27 25 PM

### CODE REVIEW

File: src/main/webapp/feedback.jsp [hunk 1]
```diff
@@ -16,35 +16,77 @@
...
         <h4 class="text-center">Please submit your feedback for your interviewer below.</h4>
         <h5 class="text-center lead">Please enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree).</h5>
-        <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewer-feedback" class="needs-validation" novalidate>
+        <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewer-feedback" class="needs-validation" onkeydown="return event.key != 'Enter';" novalidate>
           <div class="form-group">
             <div class="form-group form-inline"> 
               <label for="question1">I was comfortable during the interview:</label><br>
-              <input type="number" class="form-control" min="1" max="10" id="question1" name="question1" required><br>
+              <select class="form-control" name="question1">
+                  <option>Strongly Agree</option>
```
Comment (apluscs) [inline src/main/webapp/feedback.jsp:31]: Change the instructions to match this new format. By instructions l mean
"Please enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree)."

**Review by apluscs — APPROVED** [review]

File: src/main/webapp/feedback.jsp (hunk 1 above)
Comment (GrantComm) [reply src/main/webapp/feedback.jsp:31]: Is what I have now better?

File: src/main/webapp/feedback.jsp [hunk 2]
```diff
@@ -15,36 +15,78 @@
       <c:when test= "${role == 'Interviewee'}">
         <h1 class="text-center">Feedback</h1>
         <h4 class="text-center">Please submit your feedback for your interviewer below.</h4>
-        <h5 class="text-center lead">Please enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree).</h5>
-        <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewer-feedback" class="needs-validation" novalidate>
+        <h5 class="text-center lead">Please select an option for each prompt.</h5>
+        <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewer-feedback" class="needs-validation" onkeydown="return event.key != 'Enter';" novalidate>
```
Comment (meagancheese) [inline src/main/webapp/feedback.jsp]: Long line

File: src/main/webapp/feedback.jsp [hunk 3]
```diff
@@ -63,36 +105,78 @@
           <c:when test= "${role == 'Interviewer'}">
             <h1 class="text-center">Feedback</h1>
             <h5 class="text-center">Please submit your feedback for your interviewee below.</h4>
-            <h5 class="text-center lead">Please enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree)</h5>
-            <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewee-feedback" class="needs-validation" novalidate>
+            <h5 class="text-center lead">Please select an option for each prompt.</h5>
+            <form name="feedbackForm" method="POST" onSubmit="addScheduledInterviewId()" action="/interviewee-feedback" class="needs-validation" onkeydown="return event.key != 'Enter';" novalidate>
```
Comment (meagancheese) [inline src/main/webapp/feedback.jsp]: Long line

**Review by meagancheese — APPROVED** [review]
Some long lines, otherwise LGTM.

File: src/main/webapp/feedback.jsp (hunk 2 above)
Comment (GrantComm) [reply src/main/webapp/feedback.jsp]: Done

File: src/main/webapp/feedback.jsp (hunk 3 above)
Comment (GrantComm) [reply src/main/webapp/feedback.jsp]: Done

**Review by acmcarther — APPROVED** [review]
Does this work without changes to the backend? I'm surprised if that's the case.

**Comment by GrantComm** [comment]
> Does this work without changes to the backend? I'm surprised if that's the case.

Yes, it still works without changes to the backend because it is still strings being sent from the form. I have included a screenshot of the email in the description

**Pull request merged**